### PR TITLE
add missing metad doc

### PR DESCRIPTION
**Pull request by InnocentBug**
There seems to be a problem with duplicate documenting bias.
But I don't understand the error message.

### CODE REVIEW

File: pysages/methods/metad.py [hunk 1]
```diff
@@ -20,7 +20,9 @@
 
 class MetadynamicsState(NamedTuple):
     """
-    Attributes
+    Helper Metadynamics state
+
+    Parameters
```
Comment (sivadasetty) [inline pysages/methods/metad.py:25]: It looks like changing `Parameters` to `Arguments` might fix the duplicate warning issue. All the other changes done below are probably not needed?

File: docs/source/module-pysages-methods.rst [hunk 2]
```diff
@@ -11,6 +11,7 @@ Available Methods
    pysages.methods.ann.ANN
    pysages.methods.ffs.FFS
    pysages.methods.funn.FUNN
+   pysages.methods.metad.Metadynamics
    pysages.methods.umbrella_integration.UmbrellaIntegration
    pysages.methods.harmonic_bias.HarmonicBias
```
Comment (sivadasetty) [inline docs/source/module-pysages-methods.rst:17]: Could you also add `pysages.methods.utils.MetaDLogger` under,

.. autosummary::

   pysages.methods.utils.HistogramLogger
   **pysages.methods.utils.MetaDLogger**

File: pysages/methods/metad.py (hunk 1 above)
Comment (InnocentBug) [reply pysages/methods/metad.py:25]: Yes, the `Parameters` vs `Arguments` fixes the duplicate issue.

The other changes are needed to document `Metadynamics` correctly.
The documentation goes before the `__init__` not after it.

File: docs/source/module-pysages-methods.rst (hunk 2 above)
Comment (InnocentBug) [reply docs/source/module-pysages-methods.rst:17]: sure, done.

File: pysages/methods/metad.py [hunk 3]
```diff
@@ -20,7 +20,9 @@
 
 class MetadynamicsState(NamedTuple):
     """
-    Attributes
+    Helper Metadynamics state
```
Comment (pabloferz) [inline pysages/methods/metad.py:23]:
```suggestion
    Metadynamics helper state
```

**Review by pabloferz — APPROVED** [review]

**Pull request merged**